src/coffee/export.coffee

```diff
@@ -182,7 +192,7 @@ class Export
       Promise.map productTypes.body.results, (type) =>
         console.log 'Processing products with productType "%s"', type.name
         csv = new ExportMapping().createTemplate(type, [lang])
-        fileName = _.slugify(type.name)+"_"+type.id+".csv"
+        fileName = _.slugify(type.name)+"_"+type.id+"."+@options.exportFormat
```

Comment:
This code is used when doing a full export. The problem there is, that different products (with different productTypes) can have different CSV columns so they have to be saved in separated files (and there archived together). This feature is already published so again as in a previous comment - if you agree, we can add "templating system" for exported fileNames in next PR. Main purpose of this PR is to allow user to do a XLSX export.